### PR TITLE
[AI Chat]: Update styling on suggestions

### PR DESCRIPTION
**Pull request by fallaciousreasoning**
Resolves https://github.com/brave/brave-browser/issues/42107

## Submitter Checklist:

- [ ] I confirm that no [security/privacy review is needed](https://github.com/brave/brave-browser/wiki/Security-reviews) and no other type of reviews are needed, or that I have [requested](https://github.com/brave/reviews/issues/new/choose) them
- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Squashed any review feedback or "fixup" commits before merge, so that history is a record of what happened in the repo, not your PR
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally:
  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Comment by petemill** [comment]
@fallaciousreasoning so this hardcodes a static emoji? Will having a different emoji for each of the (also static) suggestions for non-content-associated conversations be a follow-up?

**Comment by fallaciousreasoning** [comment]
@petemill just had a quick look and the design file doesn't reference which icons should show for these. I think lets do it in a follow up.

cc @aguscruiz if you could provide the default unicode/Nala icon for the static starter prompts that'd be great!

File: components/ai_chat/resources/page/components/suggested_question/style.module.scss [hunk 1]
```diff
@@ -3,9 +3,35 @@
...
-  text-align: left;
-  font: var(--leo-font-default-regular);
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--leo-radius-s);
+  flex-shrink: 0;
 }
+
+.buttonText {
+    display: flex;
```
Comment (petemill) [inline components/ai_chat/resources/page/components/suggested_question/style.module.scss:32]: we've got the mix between 2 spaces and 4 again...

**Review by petemill — APPROVED** [review]
Please normalize indentation to 2 spaces before merging 😄

**Comment by aguscruiz** [comment]
> @petemill just had a quick look and the design file doesn't reference which icons should show for these. I think lets do it in a follow up.
> 
> cc @aguscruiz if you could provide the default unicode/Nala icon for the static starter prompts that'd be great!

What are our static prompts gonna be? Or just one in general you mean? 

If it's a default we can use as catch-all, then the 💬 emoji should work

**Comment by fallaciousreasoning** [comment]
The static prompts are the ones listed here:
https://docs.google.com/document/d/1vpyqa38tQh74mTmyr0ggcaFx1znNkjubvhB7rk3L-Wk/edit?tab=t.0#heading=h.62l2ojf8aw0t

(under `no context`)

File: components/ai_chat/resources/page/components/suggested_question/style.module.scss (hunk 1 above)
Comment (fallaciousreasoning) [reply components/ai_chat/resources/page/components/suggested_question/style.module.scss:32]: Man, I have no idea why this keeps happening :D

**Comment by aguscruiz** [comment]
@fallaciousreasoning I added a column for the emojis of each prompt in that document

**Pull request merged**